app/client/src/components/InstallChaos/index.tsx

```diff
@@ -1,57 +1,57 @@
-import { IconButton, Typography } from "@material-ui/core";
-import Done from "@material-ui/icons/DoneAllTwoTone";
-import Copy from "@material-ui/icons/FileCopyTwoTone";
-import React, { useEffect, useState } from "react";
-import { useStyles } from "./styles";
+import { IconButton, Typography } from '@material-ui/core'
+import Done from '@material-ui/icons/DoneAllTwoTone'
+import Copy from '@material-ui/icons/FileCopyTwoTone'
+import React, { useEffect, useState } from 'react'
+import { useStyles } from './styles'
 
 interface InstallProps {
-	title: string;
-	description: string;
-	yamlLink: string;
-	engine: boolean;
+	title: string
```

Comment:
Why the semicolon(;) is removed?

Reply:
I am using an auto formatter so it removes any unnecessary code.

Reply:
What formatter are you using...we used prettier and the code was consistent with semicolons everywhere...If you are using prettier use the prettierrc provided in src folder as the config file for the extension or just enable semicolons in the vscode prettier settings itself @ishangupta-ds

Reply:
I was also using prettier.

Reply:
I have updated the rc file from 4 to 2 spaces tab width and also enabled semi-colon.